crates/turborepo-lib/src/run/scope/filter.rs

```diff
@@ -75,7 +75,14 @@ impl PackageInference {
         }
 
         if selector.parent_dir != turbopath::AnchoredSystemPathBuf::default() {
-            selector.parent_dir = self.directory_root.join(&selector.parent_dir);
+            let repo_relative_parent_dir = self.directory_root.join(&selector.parent_dir);
+            let clean_parent_dir =
```

Comment:
Definitely not blocking, but we ought to think about how to work this into the path library properly